core/src/main/java/org/pentaho/di/core/database/Database.java

```diff
@@ -458,7 +458,8 @@ public void normalConnect( String partitionId ) throws KettleDatabaseException {
           String name = databaseMeta.getName();
```

Comment:
Shouldn't we get rid of this name variable? and use the datasourceName insted? 
The name variable is now only being used log if something went wrong. But shouldn't the logged connection name be dataSourceName?
Also probably put String datasourceName where the String name = databaseMeta.getName() is.

Reply:
I kept this variable because the name is a real name of datasource connection that was defined on managed datasource dialog. 
The datasourceName variable means a new datasource with variables replaced.

Reply:
Thanks. it makes sense that way.